src/client/components/Search/Search.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { Fetcher } from '../Fetcher/Fetcher';
+import { UserDetails } from '../Fetcher/UserDetails';
+import './Search.styles.css';
+
+// eslint-disable-next-line react/prop-types
+export default function Search({ getUserData }) {
+  const [search, setSearch] = useState('');
+  const [isSearching, setIsSearching] = useState(false);
+
+  const handleCancel = (e) => {
+    e.preventDefault();
+    setSearch('');
+    setIsSearching(false);
+  };
```

Comment:
No need to cancel.

Reply:
If this comment is about removing the Cancel button, I talked with Benjamin yesterday and he told that is fine if I managed a button as well. if not, could you explain a little bit more about your comment? thank you